host/src/containers/GameSessionContainer.tsx

```diff
@@ -104,20 +112,66 @@ const GameSessionContainer = () => {
     }
   }, [isModalOpen]);
 
+  useEffect(() => { //headerGame timer, 1 second interval, update localstorage in case page resets
+    if (gameTimer && !gameTimerZero){
+      let refreshIntervalId = setInterval(() => {
+        if (headerGameCurrentTime > 0) {
+          setHeaderGameCurrentTime(headerGameCurrentTime - 1);
+          localStorage.setItem('currentGameTimeStor', headerGameCurrentTime-1);
+        }
+        else
+          setGameTimerZero(true);
+      }, 1000);
+      return () => clearInterval(refreshIntervalId);
+    }
+  }, [gameTimer, gameTimerZero, headerGameCurrentTime]);
+
+  useEffect(()=>{ //update gameSession currentTimer every 3 seconds with time from headerGame Timer 
```

Comment:
Same nit as above in addition to space before and after `=>`

Reply:
commenting updated and spaces after `=> `added